### PR TITLE
Show the number of instances with connectivity problems on the monitoring overview screen

### PR DESCRIPTION
**Pull request by ramonsmits**
image

Depends on https://github.com/Particular/ServiceControl/pull/1989

### CODE REVIEW

File: src/ServicePulse.Host/ServicePulse.Host.csproj [hunk 1]
```diff
@@ -36,4 +36,10 @@
     <EmbeddedResource Include="Hosting\Help.txt" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/ServicePulse.Host.csproj:39]: Does this need to be explicitly added? I think it should automatically get embedded based on line 31.

File: src/ServicePulse.Host/app/modules/monitoring/js/monitored_endpoints.controller.js [hunk 2]
```diff
@@ -81,7 +81,7 @@
                     if (endpoint.error) {
                         connectivityNotifier.reportFailedConnection();
                         if ($scope.endpoints) {
-                            $scope.endpoints.forEach((item) => item.isScMonitoringDisconnected = true);
+                            $scope.endpoints.forEach((item) => {item.isScMonitoringDisconnected = true;item.disconnectedCount=10;});
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/modules/monitoring/js/monitored_endpoints.controller.js:84]:
```suggestion
                            $scope.endpoints.forEach((item) => {
                                item.isScMonitoringDisconnected = true;
                                item.disconnectedCount=10;
                            });
```

File: src/ServicePulse.Host/app/modules/monitoring/views/monitored_endpoints.html [hunk 3]
```diff
@@ -81,9 +81,14 @@ <h1>Endpoints overview</h1>
                                         <span class="warning" ng-if="endpoint.isScMonitoringDisconnected">
                                             <i class="fa pa-monitoring-lost endpoints-overview" uib-tooltip="Unable to connect to monitoring server"></i>
                                         </span>
-                                        <span class="warning" ng-if="endpoint.isStale">
+                                        <span class="warning" ng-if="endpoint.isStale && (endpoint.connectedCount==null || endpoint.connectedCount==0)">
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/modules/monitoring/views/monitored_endpoints.html:84]:
```suggestion
                                        <span class="warning" ng-if="endpoint.isStale && endpoint.connectedCount">
```

Type coalescing in JavaScript will convert the check to:

* Does endpoint.connectedCount have a defined value?
* Is the value non-null?
* Is the value non-zero?
* Is the value a non-empty string?

**Comment by WilliamBZA** [comment]
This does mean that if the user is on an old version of SC, they won't see the current warning icon at all.

File: src/ServicePulse.Host/ServicePulse.Host.csproj (hunk 1 above)
Comment (ramonsmits) [reply src/ServicePulse.Host/ServicePulse.Host.csproj:39]: @WilliamBZA strange.. that must be removed....Pretty sure I did that

**Pull request merged**